src/h5web/visualizations/heatmap/models.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  interpolateCool,
+  interpolateMagma,
+  interpolateInferno,
+  interpolatePlasma,
+  interpolateViridis,
+  interpolateBrBG,
+  interpolatePRGn,
+  interpolatePiYG,
+  interpolatePuOr,
+  interpolateRdBu,
+  interpolateRdGy,
+  interpolateRdYlBu,
+  interpolateRdYlGn,
+  interpolateSpectral,
+  interpolateBlues,
+  interpolateGreens,
+  interpolateGreys,
+  interpolateOranges,
+  interpolatePurples,
+  interpolateReds,
+  interpolateWarm,
+  interpolateBuGn,
+  interpolateBuPu,
+  interpolateGnBu,
+  interpolateOrRd,
+  interpolatePuBuGn,
+  interpolatePuBu,
+  interpolatePuRd,
+  interpolateRdPu,
+  interpolateYlGnBu,
+  interpolateYlGn,
+  interpolateYlOrBr,
+  interpolateYlOrRd,
+  interpolateRainbow,
+  interpolateSinebow,
+} from 'd3-scale-chromatic';
+
+export const COLOR_MAPPINGS: Record<string, (t: number) => string> = {
```

Comment:
You could add a type for interpolator functions. You could then use it in your `ColorBar` component.